### PR TITLE
sipsorcery gitian key

### PR DESCRIPTION
**Pull request by sipsorcery**

### CODE REVIEW

File: contrib/gitian-keys/keys.txt [hunk 1]
```diff
@@ -30,3 +30,4 @@ ED9BDF7AD6A55E232E84524257FF9BDBCC301009 Sjors Provoost
 AEC1884398647C47413C1C3FB1179EB7347DC10D Warren Togami
 79D00BAC68B56D422F945A8F8E3A8F3247DBCBBF Willy Ko
 71A3B16735405025D447E8F274810B012346C9A6 Wladimir J. van der Laan
+9D3CC86A72F8494342EA5FD10A41BDC3F4FAFF1C Aaron Clauson (sipsorcery)
```
Comment (hebasto) [inline contrib/gitian-keys/keys.txt]: It should be in the first name alphabetic order.

Comment (sipsorcery) [reply contrib/gitian-keys/keys.txt]: Ahh somehow didn't spot that. Fixed now. Woo hoo straight to the top of the list (if accepted that is)!

**Review by fanquake — COMMENTED** [review]
Concept ACK 906391101754f0d5188b561ff820fa4e7784d634

```
gpg --recv-key 9D3CC86A72F8494342EA5FD10A41BDC3F4FAFF1C
gpg: key 0A41BDC3F4FAFF1C: no user ID
gpg: Total number processed: 1
```

```bash
gitian.sigs $ gpg --verify 0.18.1-linux/sipsorcery/bitcoin-linux-0.18-build.assert.sig 0.18.1-linux/sipsorcery/bitcoin-linux-0.18-build.assert
gpg: Signature made Sat 10 Aug 09:45:44 2019 AWST
gpg:                using RSA key 9D3CC86A72F8494342EA5FD10A41BDC3F4FAFF1C
gpg: key 0A41BDC3F4FAFF1C: no user ID
gpg: Total number processed: 1
gpg: Can't check signature: No public key
```

Has been [producing sigs for most of the year](https://github.com/bitcoin-core/gitian.sigs/commits?author=sipsorcery) and has now signed for 3 major releases. 

@sipsorcery Is there any particular reason you haven't been building and commiting the Windows signed sigs?

I'd also have no issue with adding `sipsorcery` to the [bitcoin/bitcoin org](https://github.com/orgs/bitcoin/people), as he and `NicolasDorier` are basically taking care of anything Windows related.

**Comment by sipsorcery** [comment]
@fanquake I had assumed the Windows signed sigs needed the private code signing key. I guess that's not the case? I'll take a look at the docs again.

**Comment by laanwj** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/16581/commits/906391101754f0d5188b561ff820fa4e7784d634

**Comment by laanwj** [comment]
> @fanquake I had assumed the Windows signed sigs needed the private code signing key. I guess that's not the case? I'll take a look at the docs again.

No, that's not the case, the signing stage is really "attach  existing signature to known output" not so much "sign"

**Pull request merged**